htu21d_async.py

```diff
@@ -0,0 +1,84 @@
+from machine import I2C, Pin
+import uasyncio as asyncio
+
+class HTU21D(object):
```

Comment:
why do you copy the complete class instead of inherit it?

Reply:
I tested adding async functions to the original library first but my tests showed me that I could save RAM by copying the class and removing the sync functions, so I went for that way.